tests/cypress/integration/admin.test.js

```diff
@@ -1,8 +1,14 @@
 import {deactivatePlugin} from "@10up/cypress-wp-utils/lib/commands/deactivate-plugin";
 
 describe('Admin can login and make sure plugin is activated', () => {
+    before(() => {
+        cy.login();
+    });
+
     it('Can activate plugin if it is deactivated', () => {
-        cy.activatePlugin('simple-local-avatars');
-        cy.deactivatePlugin('simple-local-avatars');
+        cy.visitAdminPage("plugins.php");
+        cy.get("#deactivate-simple-local-avatars, tr[data-slug=simple-local-avatars] .deactivate a").click();
+        cy.get("#activate-simple-local-avatars, tr[data-slug=simple-local-avatars] .activate a").click();
+        cy.get("#deactivate-simple-local-avatars, tr[data-slug=simple-local-avatars] .deactivate a").should("be.visible");
```

Comment:
We should utilize the Cypress Utils Library and use the `activatePlugin` and `deactivatePlugin` commands.

* https://github.com/10up/cypress-wp-utils/blob/trunk/src/commands/activate-plugin.ts
* https://github.com/10up/cypress-wp-utils/blob/trunk/src/commands/deactivate-plugin.ts